src/main/java/com/gregtechceu/gtceu/api/block/MaterialBlock.java

```diff
@@ -276,4 +280,28 @@ public boolean replaceWithFramedPipe(Level level, BlockPos pos, BlockState state
         }
         return false;
     }
+
+    @Override
+    public void entityInside(BlockState state, Level level, BlockPos pos, Entity entity) {
+        if (this.tagPrefix == TagPrefix.frameGt && entity instanceof LivingEntity livingEntity) {
```

Comment:
tbh this should be in its own block class so it isn't hardcoded to frames but I'll fix that later